Makefile

```diff
@@ -23,6 +23,10 @@ run-link-checker:
 
 check-links-ci: set-up-link-checker run-link-checker
 
+.PHONY: sdkexamples
+sdkexamples:
+	cd sdkexamples; make
```

Comment:
In case you didn't know `make -C sdkexamples` would work too :)

Reply:
done!